app/models/dish.rb

```diff
@@ -18,16 +18,24 @@ class Dish < ActiveRecord::Base
   RATE_DIMENSION_DISH = 'dish'
   ratyrate_rateable RATE_DIMENSION_DISH
 
-  def rates
+  # public activity
+  include PublicActivity::Model
+  tracked owner: Proc.new{ |controller, model| controller.try(:current_user) }, recipient: Proc.new{ |controller, model| model.try(:user) } 
```

Comment:
I suspect this might work, though I don't think its an improvement - just adding it here as a curiosity:

```ruby
tracked owner: :current_user.to_proc, recipient: :user 
```